README.md

```diff
@@ -188,96 +188,12 @@ We love **Windows** too! Please refer to [Building on Windows](https://github.co
 Drafter bindings in other languages:
 
 - [Protagonist](https://github.com/apiaryio/protagonist) (Node.js)
+- [drafter.js](https://github.com/apiaryio/drafter.js) (Pure JavaScript)
```

Comment:
I know, technically it's not a binding, but I think it makes sense to list it here regardless.